### ISSUE TITLE
speech demos issues

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-demos-node#42]
As discussed with @zorkow.

Besides #41 there are other issues with the demos

* container is striped by default - but the container has the aria-label
* the container lacks a suitable role - custom elements have role=generic which does not allow for aria-label. With a flat label approach of these demos, I tend to go with `img` for simplicity's sake.

I also had issues with xml validation but I don't recall if that was a problem with the demo or with the jsdom/litedom adaptors.

**Comment by zorkow** [comment]
#49 should fix these things. Even for deep speech you get `aria-label` and `role` attributes. If one were to add a `tabindex=0` one could even do some rudimentary exploration, similar to hichart diagrams.

But just as a warning: These demos should show how to play with speech, not taken as fully developed a11y solutions.
